diff --git a/lib/inspec/dist.rb b/lib/inspec/dist.rb
index a85163b..b134da0 100644
--- a/lib/inspec/dist.rb
+++ b/lib/inspec/dist.rb
@@ -3,16 +3,16 @@
 module Inspec
   module Dist
     # When referencing a product directly, like InSpec
-    PRODUCT_NAME = "Chef InSpec"
+    PRODUCT_NAME = "Cinc-Auditor"
 
     # The inspec executable
-    EXEC_NAME = "inspec"
+    EXEC_NAME = "cinc-auditor"
 
     # The name of the server product
-    SERVER_PRODUCT_NAME = "Chef Server"
+    SERVER_PRODUCT_NAME = "Cinc Server"
 
     # name of the automate product
-    AUTOMATE_PRODUCT_NAME = "Chef Automate"
+    AUTOMATE_PRODUCT_NAME = "Cinc Dashboard"
 
     # name of the compliance product
     COMPLIANCE_PRODUCT_NAME = "Chef Compliance"
