--- inspec.rb	2019-04-12 17:01:24.000000000 +0200
+++ outspec.rb	2019-04-15 14:28:30.000000000 +0200
@@ -18,9 +18,9 @@
 
 require_relative '../../../lib/inspec/version.rb'
 
-name 'inspec'
-friendly_name 'InSpec'
-maintainer 'Chef Software, Inc <maintainers@chef.io>'
+name 'outspec'
+friendly_name 'OutSpec'
+maintainer 'Community <community@example.com>'
 homepage 'https://github.com/inspec/inspec'
 
 license 'Apache-2.0'
@@ -49,6 +49,7 @@
 dependency 'preparation'
 
 dependency 'inspec'
+dependency 'outspec'
 
 # Mark all directories world readable.
 dependency 'gem-permissions'
