diff -ur inspec/postinst outspec/postinst
--- inspec/postinst	2019-04-12 17:01:24.000000000 +0200
+++ outspec/postinst	2019-04-15 14:53:11.000000000 +0200
@@ -5,7 +5,7 @@
 #
 
 PROGNAME=`basename $0`
-INSTALLER_DIR=/opt/inspec
+INSTALLER_DIR=/opt/outspec
 CONFIG_DIR=/etc/inspec
 USAGE="usage: $0"
 
@@ -27,7 +27,7 @@
     PREFIX="/usr"
 fi
 
-binaries="inspec"
+binaries="outspec"
 for binary in $binaries; do
   rm -f $PREFIX/bin/$binary
 done
@@ -36,11 +36,19 @@
   ln -sf $INSTALLER_DIR/bin/$binary $PREFIX/bin || error_exit "Cannot link $binary to $PREFIX/bin"
 done
 
+wrapper_links="inspec"
+link_target="outspec-wrapper"
+for link in $wrapper_links; do
+  if [ ! -e $PREFIX/bin/$link ]; then
+    ln -sf $INSTALLER_DIR/bin/$link_target $PREFIX/bin/$link || error_exit "Cannot link $link_target to $PREFIX/bin/$link"
+  fi
+done
+
 # Ensure all files/directories in $INSTALLER_DIR are owned by root. This
 # has been fixed on new installs but upgrades from old installs need to
 # be manually fixed.
 chown -Rh 0:0 $INSTALLER_DIR
 
-echo "Thank you for installing InSpec!"
+echo "Thank you for installing OutSpec!"
 
 exit 0
diff -ur inspec/postrm outspec/postrm
--- inspec/postrm	2019-04-12 17:01:24.000000000 +0200
+++ outspec/postrm	2019-04-15 15:29:43.000000000 +0200
@@ -4,6 +4,8 @@
 # after package is uninstalled.
 #
 
+INSTALLER_DIR=/opt/outspec
+
 is_darwin()
 {
   uname -v | grep "^Darwin" 2>&1 >/dev/null
@@ -16,10 +18,18 @@
 fi
 
 cleanup_symlinks() {
-  binaries="inspec"
+  binaries="outspec"
   for binary in $binaries; do
     rm -f $PREFIX/bin/$binary
   done
+
+  wrapper_links="inspec"
+  link_target="outspec-wrapper"
+  for link in $wrapper_links; do
+    if [ -L $PREFIX/bin/$link -a "$(readlink $PREFIX/bin/$link)" = "$INSTALLER_DIR/bin/$link_target" ]; then
+      rm -f $PREFIX/bin/$link
+    fi
+  done
 }
 
 # Clean up binary symlinks if they exist
@@ -32,6 +42,6 @@
   cleanup_symlinks
 fi
 
-echo "InSpec has been uninstalled!"
+echo "OutSpec has been uninstalled!"
 
 exit 0
diff -ur inspec/preinst outspec/preinst
--- inspec/preinst	2019-04-12 17:01:24.000000000 +0200
+++ outspec/preinst	2019-04-12 17:01:28.000000000 +0200
@@ -4,4 +4,4 @@
 # before package is installed.
 #
 
-echo "You're about to install InSpec!"
+echo "You're about to install OutSpec!"
