From 994d36b34450d4a1fe2c9f6507f8059072ab7577 Mon Sep 17 00:00:00 2001
From: Lance Albertson <lance@osuosl.org>
Date: Mon, 23 Sep 2019 21:34:59 -0700
Subject: [PATCH 1/2] Cinc stable patches

---
 lib/inspec/dist.rb | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/lib/inspec/dist.rb b/lib/inspec/dist.rb
index a85163b3..ac9be6a0 100644
--- a/lib/inspec/dist.rb
+++ b/lib/inspec/dist.rb
@@ -3,16 +3,16 @@
 module Inspec
   module Dist
     # When referencing a product directly, like InSpec
-    PRODUCT_NAME = "Chef InSpec"
+    PRODUCT_NAME = "Cinc Auditor"
 
     # The inspec executable
-    EXEC_NAME = "inspec"
+    EXEC_NAME = "cinc-auditor"
 
     # The name of the server product
-    SERVER_PRODUCT_NAME = "Chef Server"
+    SERVER_PRODUCT_NAME = "Cinc Server"
 
     # name of the automate product
-    AUTOMATE_PRODUCT_NAME = "Chef Automate"
+    AUTOMATE_PRODUCT_NAME = "Cinc Dashboard"
 
     # name of the compliance product
     COMPLIANCE_PRODUCT_NAME = "Chef Compliance"
-- 
2.20.1

